net/rs-aggregate/Portfile

```diff
@@ -0,0 +1,219 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cargo 1.0
+
+github.setup        ktims rs-aggregate v0.3.2
+github.tarball_from archive
+categories          net
+platforms           darwin
```

Comment:
`platforms darwin` is implicit in MacPorts now.  You can remove this line.

Reply:
Will do, thanks!